ntoskrnl/ke/amd64/trap.S

```diff
@@ -1105,6 +1104,37 @@ KiSwitchKernelStackHelper:
     mov rax, rdx
     ret
 
+EXTERN KiSwitchKernelStack:PROC
+
+PUBLIC KeSwitchKernelStack
+FUNC KeSwitchKernelStack
```

Comment:
About your commit comment, isn't then it problematic in 32-bit code that we use the "old" way (without all this ASM)?